rdkit/VLib/Filter.py

```diff
@@ -3,10 +3,10 @@
 #  Copyright (C) 2003 Rational Discovery LLC
 #     All Rights Reserved
 #
-import sys
+
+import six
```

Comment:
All changes reversed. I wonder if there is a more obvious way to document the changes to the original six module. I will think about this and propose an alternative if I find a reasonable one.